src/game.py

```diff
@@ -78,27 +93,119 @@ def duel(
                 + f"({chosen_spell['Description']})!"
             )
             print(
-                f"{character['Name']} has a total power rating of {battle_stats[i]}!\n"
+                f"{character['Name']} has a total power"
```

Comment:
Does this power score include the score form a spell?

Reply:
yes, because it's coming from `battle_stats[i]`